### ISSUE TITLE
Some questions regarding the dataset details

### ISSUE THREAD

**Issue by XuelianCheng** [med-air/EndoNeRF#10]
Hi,

Thanks for sharing this great work! I have several questions regarding the dataset.

1.  I am wondering what is the baseline of the stereo cameras?  I suppose the baseline is 8mm if your dataset is collected from the Da Vinci Xi instrument. Does the unit of depth equal to millimetres(mm)?
2.  The depth map is black in several frames, i.e ''frame-000021.depth.png", "frame-000038.depth.png".. in the scene'cutting_tissues_twice'.  Any reason for the missing depth value in these frames?
3. How to compute the accurate focal length, which is provided as 569.4682 in the file 'inpose_bounds.npy'? The stereo input images look like being cropped. 

Please correct me if I am wrong!

**Comment by yuehaowang** [comment]
Thank you very much for your good questions.

1. The baseline is 4.324mm in our setting. Yes, the depths are in millimeters.
2. I guess the black depth map is caused by corrupt depth estimation, e.g., some estimated depths are wrongly super large. In this way, normally estimated depths are relatively much smaller, thereby looking black. If you read the mentioned two depth maps into arrays, you'll find the pixel values are not constantly zeros.
3. The focal length was estimated by camera calibration. If I remembered correctly, we simply employed the OpenCV toolbox to do the calibration. And yes, we crop out the console GUIs.

Hope my reply is helpful.
